src/averageAges.js

```diff
@@ -20,6 +20,22 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const men = people.filter(person => person.sex === 'm');
+
+  const menInCentury = century
+    ? men.filter(person => (
+      Math.ceil(person.died / 100) === century
+    ))
+    : men;
+
+  const sumOfAge = menInCentury.reduce((sum, person) => (
+    sum + (person.died - person.born)
+  ), 0);
```

Comment:
for convenience you can create a separate function to calculate the average of people age, because you'r repeating yourself here, and in 68 line.

for example:

```
function findAvg(people) {
  const avgAge = people.reduce((x, year) => x + (year.died - year.born), 0);

  return avgAge / people.length;
}
```

and call this function inside `calculateMenAverageAge` and `calculateWomenAverageAge`

Reply:
yes, great advice, thank you!